packages/web/src/scss/tools/_typography.scss

```diff
@@ -1,46 +1,63 @@
 @use 'sass:map';
+@use 'sass:meta';
 @use 'sass:string';
 @use 'breakpoint';
 @use '@tokens' as tokens;
+@use 'string' as spirit-string;
+
+// Typography properties supported by mixins in this file.
+// Matches `TypographyShape` in Spirit Tokens Exporter.
+$_properties: (font-style, font-weight, font-size, line-height, font-family);
+
+// Mixin to generate typography styles
+// Parameters are:
+// * $token: the composite token to generate styles from
+// * $as-custom-properties: if true, generates custom properties instead of CSS properties
+// * $infix: infix to use for custom properties (e.g. 'input' for input fields)
```

Comment:
maybe this prop name should be clear it is only for the custom property? 
`$custom-property-infix`